### PR TITLE
[DOC] update the turning guide document issues [skip ci]

### PR DESCRIPTION
**Pull request by nvliyuan**
This pr is to address some document issues: #9026
Signed-off-by: liyuan <yuali@nvidia.com>

### CODE REVIEW

File: docs/tuning-guide.md [hunk 1]
```diff
@@ -101,7 +102,7 @@ the spill storage, so it can use pinned memory too.
 
 Configuration key: [`spark.rapids.memory.host.spillStorageSize`](configs.md#memory.host.spillStorageSize)
 
-Default value: `1g`
+Default value: `-1`
```
Comment (jlowe) [inline docs/tuning-guide.md:105]: Without proper context this default value doesn't make sense with the description below.  Part of the issue here is we're documenting a configuration in two places, configs.md and here.  Ideally we should describe the configs for tuning in how they're related to performance at a high-level but leave details like defaults and full description to the specific configs document that was generated (i.e.: just point to them from here).  Not something that needs to be solved for this PR, but for this specific config we should at least add a sentence explaining what a value `-1` will do, like the description of this config in configs.md does.l

Comment (nvliyuan) [reply docs/tuning-guide.md:105]: updated, could you help review again?

Comment (nvliyuan) [reply docs/tuning-guide.md:105]: Hi @jlowe , could you help review if the change is OK?

Comment (jlowe) [reply docs/tuning-guide.md:105]: I'm not seeing this comment addressed? 

> for this specific config we should at least add a sentence explaining what a value -1 will do, like the description of this config in configs.md does

**Review by jlowe — APPROVED** [review]

**Comment by nvliyuan** [comment]
build

**Comment by nvliyuan** [comment]
build

**Pull request merged**